### ISSUE TITLE
how to generate the training dataset?

### ISSUE THREAD

**Issue by anguoyang** [zcemycl/TF2DeepFloorplan#6]
the r3d.tfrecords, how to generate this file with images and labels?

**Comment by huttzza** [comment]
in original repo, [DeepFloorplan](https://github.com/zlzeng/DeepFloorplan), you can find how to make tfrecords files.
* [DeepFloorplan/create_tfrecord.py](https://github.com/zlzeng/DeepFloorplan/blob/master/utils/create_tfrecord.py)

but you can download original tfrecords in this [download link](https://github.com/zlzeng/DeepFloorplan/blob/master/dataset/download_links.txt)


> first you need img datset files,
> * [download link](https://github.com/zlzeng/DeepFloorplan/blob/master/dataset/download_links.txt)

> second you have to make file list txt file like [this format](https://github.com/zlzeng/DeepFloorplan/blob/master/dataset/r3d_train.txt)
> * [(example of mine)](https://github.com/huttzza/TF2DeepFloorplan/blob/main/data.py#L58)

> and the last, you can generate tfrecords file.
> * [write_bd_rm_record](https://github.com/zlzeng/DeepFloorplan/blob/687abf928412da39179b67612c15c02fd360d459/utils/tf_record.py#L289)

**Comment by thijser** [comment]
Unfortunatly the link to the training data is gone, 

https://mycuhk-my.sharepoint.com/:f:/g/personal/1155052510_link_cuhk_edu_hk/EgyJhisy04hNnxKncWl5zksBf9zDKDpMJ7c0V-q53_pxuA?e=P0BjZd just gives an error 404. If you still have that data it would be great if you could upload it.

Also do you happen to have the pretrained original? If it's not too much to ask it would really help me as it also gives an error 404.

**Comment by huttzza** [comment]
> Unfortunatly the link to the training data is gone,
> 
> https://mycuhk-my.sharepoint.com/:f:/g/personal/1155052510_link_cuhk_edu_hk/EgyJhisy04hNnxKncWl5zksBf9zDKDpMJ7c0V-q53_pxuA?e=P0BjZd just gives an error 404. If you still have that data it would be great if you could upload it.
> 
> Also do you happen to have the pretrained original? If it's not too much to ask it would really help me as it also gives an error 404.

@thijser oh, I didn't know the link is expired.

unfortunately, I have only r3d dataset and pretrained model, not r2d or tfrecords.
* [here is the r3d dataset download link](https://drive.google.com/file/d/1nRP2i_FIqZgOnSkHYde5U4zLMsnIm22t/view?usp=sharing)
* [pretrained model](https://drive.google.com/file/d/1y439atMPixzP296-Oa3mwoOiFJ0-4OyP/view?usp=sharing)

and tfrecords, you can make it with r3d dataset!


> > second you have to make file list txt file like [this format](https://github.com/zlzeng/DeepFloorplan/blob/master/dataset/r3d_train.txt)
> > 
> > * [(example of mine)](https://github.com/huttzza/TF2DeepFloorplan/blob/main/data.py#L58)
> 
> > and the last, you can generate tfrecords file.
> > 
> > * [write_bd_rm_record](https://github.com/zlzeng/DeepFloorplan/blob/687abf928412da39179b67612c15c02fd360d459/utils/tf_record.py#L289)

**Comment by thijser** [comment]
@huttzza thank you for the data and the model!

**Comment by thijser** [comment]
btw, if you are reading this anway, what is the best way to see how the pretrained network reacts to my input? I tried putting the pretrained network back in the original network but the demo.py there gives an a mismatch between the current graph and the graph from the checkpoint, so there is probably a small difference there.

**Comment by zcemycl** [comment]
@thijser Usually, it will be difficult to reuse pretrained network weight from different custom model, unless you can map them one by one. I have just included `deploy.py` which allows you to use the weight (`log/store/` should have two files `G.index` and `G.data-00000-of-00001` as weight) from my `train.py`.
